practice-three/src/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" href="./style/style.css" />
+ </head>
+ <body>
+   <!-- START HEADER -->
+  <header class="st">
+    <div class="header-container">
+      <div class="crossbar">
+        <div class="navbar">
```

Comment:
navbar is not suitable name for this, try [this](https://www.howtogeek.com/720214/what-is-a-hamburger-menu-button/)